README.md

```diff
@@ -138,12 +149,14 @@ Find more information about `hazelcast-member` tool at https://github.com/hazelc
 Refer to the official [Hazelcast IMDG Reference Manual](http://docs.hazelcast.org/docs/3.10.4/manual/html-single/index.html#getting-started) for more information regarding starting clusters.
```

Comment:
I think we need to be more consistent when using links to http://docs.hazelcast.org/docs/
Some usages link to `3.10.4` / `3.10.5`,  some usages refer to `latest`.

Reply:
Made them all "latest"